app/components/works/dates_component.html.erb

```diff
@@ -6,117 +6,123 @@
                                                  min_year: min_year,
                                                  max_year: max_year) %>
 
-  <div class="mb-3 row">
-    <div class="col-sm-3 h6">
-      Creation date
-      <%= render PopoverComponent.new key: 'work.creation_date' %>
-    </div>
-  </div>
+<fieldset>
```

Comment:
The indent level on this line does not match line 126. Can you make them the same?

Reply:
Done